src/Components/Aspire.StackExchange.Redis.DistributedCaching/Aspire.StackExchange.Redis.DistributedCaching.csproj

```diff
@@ -6,12 +6,19 @@
     <PackageTags>$(ComponentCachePackageTags) distributedcaching distributedcache redis</PackageTags>
     <Description>A Redis® implementation for IDistributedCache that integrates with Aspire, including health checks, logging, and telemetry.</Description>
     <PackageIconFullPath>$(SharedDir)redis-cube-red_white-rgb.png</PackageIconFullPath>
+    <IsAotCompatible>false</IsAotCompatible>
```

Comment:
I think this would not be necessary if the `Shared` code was in a custom assembly that is itself marked aot-not-compatible, right? Is it worth it?

Reply:
```suggestion
```

This should be removed. These libraries are AOT compatible.